client/src/index.js

```diff
@@ -16,27 +16,28 @@ import { BalanceReducer,
   LoginReducer,
   PaymentMethodReducer,
   RegistrationReducer,
-  SelectTipReducer,
+  TipReducer,
   TipReceiverReducer,
   TransactionReducer,
   UserReducer,
-  WithdrawalReducer, } from './Reducers';
+  WithdrawalReducer, 
+} from './Reducers';
 
 const rootReducer = combineReducers({
   BalanceReducer,
   DepositReducer,
   LoginReducer,
   PaymentMethodReducer,
   RegistrationReducer,
-  SelectTipReducer,
+  TipReducer,
   TipReceiverReducer,
   TransactionReducer,
   UserReducer,
   WithdrawalReducer,
 });
 
-const store = createStore(
-  rootReducer,
+export const store = createStore(
+  rootReducer, {},
   compose(
     applyMiddleware(thunk),
     window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
```

Comment:
Don't forget to either remove this or add a conditional when making a pull request. I think this would break the site. 👍

Reply:
Totally! Wasn't planning on deploying yet though, hence why I kept it in.